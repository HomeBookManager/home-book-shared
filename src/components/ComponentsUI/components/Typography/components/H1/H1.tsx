import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TOmittedTypographyProps } from '../../types';
import { TypographyFontType } from '../../enums';

const H1: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h1} {...restProps}>
    {children}
  </Typography>
);

export default H1;
