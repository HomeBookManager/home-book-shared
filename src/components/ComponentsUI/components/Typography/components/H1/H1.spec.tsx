import { render } from '@testing-library/react';

// components
import H1 from './H1';

describe('H1 snapshots', () => {
  it('should render H1', () => {
    //before
    const { asFragment } = render(<H1>H1</H1>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
