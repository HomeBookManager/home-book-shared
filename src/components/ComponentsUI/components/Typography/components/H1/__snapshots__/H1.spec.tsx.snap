// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 snapshots should render H1 1`] = `
<DocumentFragment>
  <h1
    class="Typography Typography--regular Typography--normal Typography--h1 Typography--without-margin"
    data-test-text=""
    style="color: rgb(30, 40, 57);"
  >
    H1
  </h1>
</DocumentFragment>
`;
